autodist/kernel/synchronization/compressor.py

```diff
@@ -205,6 +205,104 @@ class HorovodCompressorEF(CompressorEF, HorovodCompressor):  # This works becaus
     """Horovod's Compression but with Error Feedback."""
 
 
+class PowerSGDCompressor(CompressorEF):
```

Comment:
Is there a functional difference between this class and the commented one below? why not just uncomment and modify that one?

Reply:
It's the same as the commented one. I delete those comments now.

Reply:
Yeah how's your implementation diffferent from the original one by Trevin (commented below?)